app/tray/Notify/index.js

```diff
@@ -105,34 +105,6 @@ class Notify extends React.Component {
     )
   }
 
-  nonceWarning() {
-    return (
-      <div className='notifyBoxWrap' onMouseDown={(e) => e.stopPropagation()}>
-        <div className='notifyBoxSlide'>
-          <div className='notifyBox'>
-            <div className='notifyTitle'>Adjustable Nonce</div>
-            <div className='notifyBody'>
-              <div className='notifyBodyBlock notifyBodyBlockBig'>
-                <div>
-                  Adjusting the nonce of a replacement transaction will cause it to become a new transaction
-                  rather than a replacement, use with caution
```

Comment:
Are we adding this message back somewhere?

Reply:
no, not sure where to show it now that we removed the setting

Reply:
ok, feels like a follow-up ticket anyway